### DISCUSSION TITLE
Who is Marc Farmer?

### DISCUSSION THREAD

**Discussion by ghalimi** [stoic-doc/Community#23] [Documentation]
*`Definition`* `Marc Farmer`

**Comment by marc-farmer** [comment]
Marc Farmer is a whimsical character representing a typical STOIC user.

**Comment by ghalimi** [comment]
Here are some illustrations depicting our friend Marc Farmer, courtesy of [Satoshi Hashimoto](https://www.dutchuncle.co.uk/satoshi-hashimoto):

![Dreaming](https://stoic-doc.github.io/Community/images/illustrations/Dreaming.png)
![Surveying](https://stoic-doc.github.io/Community/images/illustrations/Surveying.png)
![Selecting](https://stoic-doc.github.io/Community/images/illustrations/Selecting.png)
![Nurturing](https://stoic-doc.github.io/Community/images/illustrations/Nurturing.png)
![Plowing](https://stoic-doc.github.io/Community/images/illustrations/Plowing.png)
![Planting](https://stoic-doc.github.io/Community/images/illustrations/Planting.png)
![Watering](https://stoic-doc.github.io/Community/images/illustrations/Watering.png)
![Teaching](https://stoic-doc.github.io/Community/images/illustrations/Teaching.png)
![Harvesting](https://stoic-doc.github.io/Community/images/illustrations/Harvesting.png)
![Selling](https://stoic-doc.github.io/Community/images/illustrations/Selling.png)
![Sharing](https://stoic-doc.github.io/Community/images/illustrations/Sharing.png)
